recipe/meta.yaml

```diff
@@ -49,6 +49,7 @@ outputs:
       ignore_run_exports_from:
         - {{ compiler('cuda') }}
         - libnpp-dev  # libnpp is tied to specific cuda-version at x.x, so relax it, as we support cuda-version >= 12.6, < 13
+      post-link: post-link
     files:
       - NOTICE/NOTICE
```

Comment:
This can go in the license section instead of being installed to PREFIX? Notice/notice is a very generic filename that could be clobbered by other packages.